_scripts/instruction-widget/install.js

```diff
@@ -29,10 +29,12 @@ module.exports = function(context) {
         context.distro == "sharedhost") {
         return '';
     }
-    else if ((context.distro == "debian" && context.version > 7) ||
-        (context.distro == "ubuntu" && context.version > 15.10)) {
+    else if (context.distro == "debian" && context.version > 7) {
       debian_install();
     }
+    else if (context.distro == "ubuntu" && context.version > 15.10){
```

Comment:
PPA has packages for 14.04 as well. I'd either change this to `context.version > 12.04` or `context.version >= 14.04` so the 14.04 instructions get updated.